modules/con_adding-openshift-routes-as-a-traffic-manager-for-argo-rollouts.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="adding-openshift-routes-as-a-traffic-manager-for-argo-rollouts_{context}"]
+= Adding OpenShift Routes as a traffic manager for Argo Rollouts
+
+With traffic management, a user can modify the amount of traffic to direct it to various applications in a cluster environment. Users can choose to reduce or increase the traffic based on their requirements. 
+
+By enabling routes as traffic managers, a user using OpenShift in their environment ensures that new changes roll out adequately to all the cluster applications, enabling progressive delivery.
+
+You can use Openshift Route as a plugin and automatically install the plugin with the `RolloutManager` custom resource (CR).
+
+[discrete]
+== About OpenShift routes 
+
+An link:https://docs.openshift.com/container-platform/4.14/networking/routes/route-configuration.html[OpenShift Container Platform route] exposes a service at a host name, such as `www.example.com`, so that external clients can reach it by name.
+
+Routes can be either secured or unsecured. Secure routes provide the ability to use several types of TLS termination to serve certificates to the client.
+
+To install the rollout-manager, you must:
```

Comment:
change to `To use the openshift-route rollout plugin, you must:`

Reply:
Fixed.